src/ijoin.c

```diff
@@ -34,15 +34,15 @@ SEXP lookup(SEXP ux, SEXP xlen, SEXP indices, SEXP gaps, SEXP overlaps, SEXP mul
   count = (int *)INTEGER(VECTOR_ELT(ux, uxcols-2));
   type_count = (int *)INTEGER(VECTOR_ELT(ux, uxcols-1));
   switch (mult) {
-  case FIRST:
+  case FIRST: {
     switch(type) {
-    case EQUAL:
+    case EQUAL: {
```

Comment:
I commented earlier about the `{` in case statements and was going to merge anyway as it wasn't a big deal.  But then I carried on through the PR and saw the many more changes (168) in this file (ijoin.c).  Best to undo these `{` additions then please. General rule: only use curlies please when they're needed to declare variables local to that case.